js/jquery.crellyslider.js

```diff
@@ -449,15 +452,12 @@
 			// Get the URL of the background image of the first slide
 			var img_url = getSlide(0).css('background-image');
 			img_url = img_url.replace(/^url\(["']?/, '').replace(/["']?\)$/, '');
-			
-			if(img_url == '' || img_url == 'undefined' || img_url == 'none') {
-				addPreloaderHTML();
-			}
-			else {
+
+			if (img_url) {
 				// When the background image of the first slide is loaded
 				$('<img>')
 				.load(function() {
-					addPreloaderHTML();
```

Comment:
We don't have to wait to this callback in order to create the preloader element because already exists. Just update bg img info.